src/libsystemd/sd-device/test-sd-device.c

```diff
@@ -499,6 +499,37 @@ TEST(sd_device_enumerator_add_match_parent) {
         }
 }
 
+TEST(sd_device_enumerator_add_all_parents) {
+        _cleanup_(sd_device_enumerator_unrefp) sd_device_enumerator *e = NULL;
+        int r;
+
+        assert_se(sd_device_enumerator_new(&e) >= 0);
```

Comment:
For newer tests, please use `ASSERT_OK()` macro rather than `assert_se(... >= 0);` here and below.
```c
ASSERT_OK(sd_device_enumerator_new(&e));
```